### ISSUE TITLE
How can edit the paligemma funetuning scripts to plot a validation loss curve and also convert my model to hf model 

### ISSUE THREAD

**Issue by Shaka42** [google-research/big_vision#147]
https://github.com/google-research/big_vision/blob/main/big_vision/configs/proj/paligemma/finetune_paligemma.ipynb
Can some please help ?
